gitlabform/processors/abstract_processor.py

```diff
@@ -23,6 +24,7 @@ def __init__(self, configuration_name: str, gitlab: GitLab):
             ],
         ] = {}
         self.gl: Gitlab = GitlabWrapper(self.gitlab)._gitlab
+        self.python_gitlab = PythonGitlab(self.gl)
```

Comment:
I will renamed

It could go in the wrapper probably, I wanted to add it to the Abstract so it's easy for any other converted Processors to start using the functions as it'll be directly on the Abstract

Reply:
If we want it available in the Abstract, then let's just add them to the wrapper. That'll make the code easier to read too. Otherwise we'll have probably different lines like this:

```py
self.gl.projects.members()
self.foo.get_user_id('blah')
```

Instead going with the wrapper, I think it'll look like this which I think is a bit more understandable, maybe...

```py
self.gl.projects.members()
self.gl.get_user_id('blah')
```

The above is just pseudo code. DIdn't really think it through yet.

Reply:
works for me 👍

Reply:
Not quite as simple as putting in GitlabWrapper class, because it returns an instance of `Gitlab` class as self.gl

But if I extend the `Gitlab` class we can put our convenience methods onto there and still expose in the style you've proposed above @amimas and it loks quite nice (and tbh a bit more readable the difference between `GitLab` class and `Gitlab` class, now people will see `PythonGitlab` as the type